website/content/en/highlights/2024-11-07-exclusive_route.md

```diff
@@ -0,0 +1,17 @@
+---
+date: "2024-11-07"
+title: "Exclusive Route Transform"
+description: "Introducing the exclusive route transform"
+authors: [ "binarylogic" ]
+pr_numbers: [ 21707 ]
+release: "0.43.0"
+hide_on_release_notes: false
+badges:
+  type: "new feature"
+  domains: [ "transforms" ]
+---
+
+The `ExclusiveRoute` transform splits an event stream into unique sub-streams based on user-defined conditions. Each event will only be
+routed to a single stream. This transforms complements the existing [Route transform][docs.transforms.route].
```

Comment:
I was thinking, for this, that we'd write a more complete highlight showing usage, how it differs from `route`, etc. The current highlight doesn't contain much more than the release changelog will so I don't think it is useful to add in its current form. We could delete it if you plan to write a separate blog post.

Reply:
Let's keep it, I will add an example config and a picture. And then in the blog post we can do the same for `route` and add a section explaining the differences.